engine/src/flutter/shell/platform/darwin/ios/framework/Source/UITextInput+TextRange.swift

```diff
@@ -0,0 +1,226 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+
+@objc(FlutterTextPosition)
+final class TextPosition: UITextPosition, NSCopying {
```

Comment:
discussed offline. will continue reviewing this after you make the change since you plan to update the swift code.